tests/e2e/tests/pilot/testdata/networking/v1alpha3/wildcard-tls-google.yaml

```diff
@@ -1,12 +1,14 @@
 apiVersion: networking.istio.io/v1alpha3
 kind: ServiceEntry
 metadata:
-  name: wikipedia
+  name: wildcardgoogle
```

Comment:
undo.. and use original file as is

Reply:
I believe the original file is restored as well now.